core/audits/layout-shifts.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {Audit} from './audit.js';
+import * as i18n from '../lib/i18n/i18n.js';
+import {CumulativeLayoutShift as CumulativeLayoutShiftComputed} from '../computed/metrics/cumulative-layout-shift.js';
+import CumulativeLayoutShift from './metrics/cumulative-layout-shift.js';
+import TraceElements from '../gather/gatherers/trace-elements.js';
+
+/** @typedef {LH.Audit.Details.TableItem & {node?: LH.Audit.Details.NodeValue, score: number, subItems?: {type: 'subitems', items: SubItem[]}}} Item */
+/** @typedef {{node?: LH.Audit.Details.NodeValue, url?: string, cause: LH.IcuMessage}} SubItem */
+
+const UIStrings = {
+  /** Descriptive title of a diagnostic audit that provides the top elements affected by layout shifts. */
+  title: 'Avoid large layout shifts',
```

Comment:
This is the same title as `layout-shift-elements`. I understand why since the call to action is the same, but this makes me wonder if we should somehow combine the audits (e.g. multiple tables in the same audit)

Reply:
I think I'm leaning towards the hide/remove option at this point. This new audit seems generally better than the old one and less of an information dump. That being said, if we still see value in the element summary my preferences go as follows:

1. Hide/remove the old audit
2. Add a second table to the new `layout-shifts` audit that takes the elements from the old audit
3. Make the old audit informative

Reply:
+1 to ditching the old one if possible (hidden for now to not break anyone makes sense). Is it possible to make sure all the information in the old audit is in the new audit (either improved or at least as-is)? e.g. for shifts without a known root cause

Reply:
Yeah, sorry, I commented without reading through the code or trying it out :) Sounds great to me!